### ISSUE TITLE
Identify Completely Floating Regions

### ISSUE THREAD

**Issue by diginfo** [SoftFever/OrcaSlicer#594]
It would be great if as well as this warning, it actually identifies the problem by maybe indicating it with a colour and zooming into the layer, or something similar.

I am getting this error and I cannot find where the problem is, might be that I just don't know how to see it.
<img width="796" alt="Screenshot 2023-03-27 at 10 32 44 AM" src="https://user-images.githubusercontent.com/6010010/227826839-6873734e-99f5-4eaf-840f-9c913ec56fc6.png">

<img width="398" alt="Screenshot 2023-03-27 at 10 27 54 AM" src="https://user-images.githubusercontent.com/6010010/227826348-71e2c520-e929-434c-9f3a-6c3ca28357e2.png">

**Comment by Ltek** [comment]
Agreed, I got this error today but cant find anywhere that it is floating... I'm suspecting this is a bug but if it is identifying a real issue with the design, it should colorize it so we know where.

**Comment by Ltek** [comment]
bump

**Comment by pacmac** [comment]
It's not stale, the issue still exists but is probably not a priority.

**Comment by stephbellemare** [comment]
I found that if you slice and it gives a floating warning, change the infill %, reslice and put back to original infill %, slice again, the warning will have disappeared.

**Comment by Ltek** [comment]
I find the warning has been consistent (and correct) for me... but would like the slicer to color-code the onscreen image so we can see what it has identified. Its often very difficult to see what it thinks is floating vs just a normal overhang

**Comment by ThrawnyBoy** [comment]
> I found that if you slice and it gives a floating warning, change the infill %, reslice and put back to original infill %, slice again, the warning will have disappeared.

This worked for me so definitely a bug somewhere. Thanks for the tip to workaround the error it was annoying me when you can plainly see (and _know_) there is absolutely no floating regions.

**Comment by MaDMaCKeM** [comment]
Same here, changed infill setting and the error disappeared

**Comment by Ltek** [comment]
ping

**Comment by Ltek** [comment]
not stale

**Comment by TomasL-SE** [comment]
Got this error as well, added support to se what happens, and the support ended up one layer above the top layer. 
This does not make sence

**Comment by Ltek** [comment]
still needed!

**Comment by GreavesDesignerOne** [comment]
Encountered "floating regions" error in [this IDEX calibration model](https://www.printables.com/model/201707-x-y-and-z-calibration-tool-for-idex-dual-extruder), it's a very simple model and there's no issue slicing it in Cura.

**Comment by boksbox** [comment]
Still persists at 2.4.1 beta.   

Shows warning when no visible floating regions and slice looks fine.  Prints ok.

![Image](https://github.com/user-attachments/assets/df2fceed-b1a5-4674-88d5-44cae543b760)

Warning disappears when supports enabled but no supports generated

![Image](https://github.com/user-attachments/assets/3f43dabb-04df-433c-8d6a-3b0f88786102)
